CHANGELOG.md

```diff
@@ -1,10 +1,23 @@
 # Changelog
 
+## [1.1.0](https://github.com/RedHatProductSecurity/cvelib/compare/1.0.0...1.1.0) (Nov 11, 2022)
+
+* The `publish` and `reject` subcommands have a new `-f/--cve-json-file` option that allows submitting CVE records from
+  a file (#18).
+* Added CVE v5 JSON schema (5.0.0) validation when publishing a CVE record (#39).
+* Full CVE v5 records can now be used when publishing a CVE; the CNA container is parsed from the CVE record
+  automatically (#42).
+* Automatically add `providerMetadata` from the org used when authenticating against CVE Services if it is missing in
+  the supplied CVE record (#19).
+* Added CVE v5 JSON 5.0.0 schemas under `cvelib/schemas` along with a script that extracts container-level sub-schemas.
+* `cve show --show-record --raw` now outputs a valid CVE record only (#44).
+* Dropped support for Python 3.6.
```

Comment:
LGTM aside from this one line - did we really? IIRC our use of mypy type hints is still compatible with 3.6 - we use only the builtin types like str or tuple. Either we don't use the tuple[str, int] subscripted form that requires 3.9, or maybe we import typing.Tuple and use the 3.6-compatible Tuple[str, int] form instead. Am I misremembering?

Reply:
It's not so much about the types, I was just refreshing the list of versions claimed as supported and tested against: https://github.com/RedHatProductSecurity/cvelib/commit/e70190ad712eec61ff97e6e4ae83d76b9cd1c944 . Python 3.6 has been EOL for almost a year now.